aligner/u_of_michigan_aligner/u_of_michigan_aligner.wdl

```diff
@@ -11,152 +12,167 @@
-  # so it needs more disk space.  Also it spills to disk in an uncompressed format so we need to account for that with a
-  # larger multiplier
-  Float sort_sam_disk_multiplier = 3.25
-
-
-  Float PreAlign_ref_size = if (defined(dynamically_calculate_disk_requirement)) then size(PreAlign_reference_genome_default, "GB") + size(PreAlign_reference_genome_index_default, "GB") + 
-      additional_disk else ReferenceGenome_disk_size_override_default + additional_disk
-
-  Float ref_size = if (defined(dynamically_calculate_disk_requirement)) then size(ref_fasta, "GB") + size(ref_fasta_index, "GB") + 
-      additional_disk else ReferenceGenome_disk_size_override_default + additional_disk
-
-  Float ref_extra_size = if (defined(dynamically_calculate_disk_requirement)) then size(ref_alt, "GB") + size(ref_bwt, "GB") + size(ref_pac, "GB") + 
-      size(ref_ann, "GB") + size(ref_amb, "GB") + size(ref_sa, "GB") + 
-      additional_disk else BWT_disk_size_override_default + additional_disk
-
-  Float dbsnp_size =if (defined(dynamically_calculate_disk_requirement)) then size(dbSNP_vcf, "GB") + size(dbSNP_vcf_index, "GB") + 
-     additional_disk else dbSNP_disk_size_override_default + additional_disk
-
-  Float cram_and_crai_size = if (defined(dynamically_calculate_disk_requirement)) then size(input_cram_file, "GB") + size(input_crai_file, "GB") + 
-     additional_disk else CRAMandCRAI_disk_size_override_default + additional_disk
-
-  Float fastq_gz_files_size = CRAM_to_fastqgz_multiplier * cram_and_crai_size
-
-
-
-  Float PreAlign_disk_size = PreAlign_ref_size + (bwa_disk_multiplier * cram_and_crai_size) + 
-     (sort_sam_disk_multiplier * cram_and_crai_size) + cram_and_crai_size + additional_disk + fastq_gz_files_size
-
-  Float Align_disk_size = ref_size + ref_extra_size + (bwa_disk_multiplier * fastq_gz_files_size) + additional_disk
-
-  # The merged cram can be bigger than the summed sizes of the individual aligned crams,
-  # so account for the output size by multiplying the input size by bwa disk multiplier.
-  Float PostAlign_disk_size = ref_size + dbsnp_size + cram_and_crai_size + 
-     (sort_sam_disk_multiplier * cram_and_crai_size) + (bwa_disk_multiplier * cram_and_crai_size) + additional_disk
-
+  input {
+
+    # The first nine variables are neccessary to convert a float to int
+    # https://gatkforums.broadinstitute.org/wdl/discussion/9541/convert-float-to-int
+    # The old version, disks: "local-disk " + sub(disk_size, "\\..*", "") + " HDD"
+    # being set as as in input for the task's inputs, previously worked but fails to 
+    # pass womtool in WDL 1.0 format; documentation on this change is absent
+
+    String PreAlign_disk_size_string = PreAlign_disk_size
+    String PreAlign_disk_size_before_decimal = sub(PreAlign_disk_size_string, "\\..*", "")
+    Int PreAlign_disk_size_int = PreAlign_disk_size_before_decimal
```

Comment:
You can convert from a Float to an Int by using ceil if you want, as in  disks: "local-disk " + ceil(disk_size) + " HDD"
https://github.com/DataBiosphere/topmed-workflow-variant-calling/blob/2bc37748e11d04f5d5a3ee69029947fce7c55aee/variant-caller-wdl/topmed_freeze8_caller.wdl#L543